api/searchCases/index.ts

```diff
@@ -1,137 +1,171 @@
-import { caseMetaDataFiledsMapping } from '../configuration/mappings'
-import { fieldNameMapper } from '../lib/util'
+import {UserInfo} from '../auth/interfaces/UserInfo'
+import {getConfigValue} from '../configuration'
+import {caseMetaDataFiledsMapping} from '../configuration/mappings'
+import {WILDCARD_SEARCH_FIELDS, WILDCARD_SEARCH_ROLES} from '../configuration/references'
+import {fieldNameMapper} from '../lib/util'
+import {ElasticSearchQuery} from './interfaces/ElasticSearchQuery'
 
 /**
  * Manually creating Elastic search query
  */
 export function modifyRequest(proxyReq, req) {
-    const request = prepareElasticQuery(req.query, req.body)
+  const userInfo = req.session.passport.user.userinfo as UserInfo
```

Comment:
A lack of a semicolon is a lint warning. These are continuing to build up in the code base.